### ISSUE TITLE
pressing 1 crashes even though the json is in the folder (dialogs/)

### ISSUE THREAD

**Issue by frrozenjr** [FigmentBoy/DialogHandler#3]

**Comment by SuperLucasGamer** [comment]
well actually you see, turns out the creator did a oopsie and put a comma (,) on the last speech
ok, let me rephrase that, basically it starts by covering up all of the code, then it sets up the color and then it adds a new covery-thingy whatever idk code, which has all of the lines, then it has a covery thingy fo revery line, with a comma before the new one starts (right after the "}") however the creator made an oopsie and added the comma to the last speech, and sence theres not another speech, it goes oof oof. so remove that comma and it works, i think..idk..
I mean, i hope it will. im kinda new to github n all this..
